draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -538,7 +509,7 @@ Procedure:
 8.  B = P1 + Q_1 * domain + H_1 * msg_1 + ... + H_L * msg_L
 9.  Abar = A * r1
 10. Bbar = B * r1 - Abar * e
-11. C = Bbar * r2 + Abar * r3 + H_j1 * m~_j1 + ... + H_jU * m~_jU
+11. T =  Abar * r3 + Bbar * r2 + H_j1 * m~_j1 + ... + H_jU * m~_jU
 12. c = calculate_challenge(Abar, Bbar, C, (i1, ..., iR),
```

Comment:
Change `C` to `T` since it's been renamed.

Reply:
Fixed. Thanks!